x/wasm/keeper/msg_dispatcher.go

```diff
@@ -155,6 +159,15 @@ func (d MessageDispatcher) DispatchSubmessages(ctx sdk.Context, contractAddr sdk
 	return rsp, nil
 }
 
+func redactError(err error) string {
+	// FIXME: do we want to hardcode some constant string mappings here as well?
+	// Or better document them? (SDK error string may change on a patch release to fix wording)
+	// sdk/11 is out of gas
+	// sdk/5 is insufficient funds (on bank send)
+	codespace, code, _ := sdkerrors.ABCIInfo(err, false)
+	return fmt.Sprintf("codespace: %s, code: %d", codespace, code)
```

Comment:
I think this is a pretty nice error string. Can be parsed if really needed but does not screem PARSE ME.